### PR TITLE
Add blogpost about monorepositories

### PR DESCRIPTION
**Pull request by lbarcziova**
Related to #642

### CODE REVIEW

File: content/posts/monorepos.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+## Wrapping up
+
+If you have a repository in the monorepo format, give it a try now and share your feedback with us!
+
+Going forward, we are about to implement the [possibility of defining dependencies between jobs](https://github.com/packit/packit-service/issues/2105),
+allowing for even greater flexibility and control of monorepository jobs.
+
+We are also happy to help with any contributions from the community to help us expand and refine the support with additional functionalities,
+such as [being able to define paths the service should react to](https://github.com/packit/packit-service/issues/2006)
+or [templating features](https://github.com/packit/packit/issues/1925).
```
Comment (lbarcziova) [inline content/posts/monorepos.md:101]: do you have any more suggestions on what else to include?

File: content/posts/monorepos.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    paths:
+      - ./backend
+    specfile_path: copr-backend.spec
+    files_to_sync:
+      - copr-backend.spec
+```
+
+That was the configuration of the packages and their locations in general, but how should these values then be utilised?
+
+### Packit Service support
```
Comment (lachmanfrantisek) [inline content/posts/monorepos.md:57]: What about adding a screenshot of the status to make it more eye-cathing (we can also use the same for Mastodon post that can market this post).

File: content/posts/monorepos.md (hunk 1 above)
Comment (lachmanfrantisek) [reply content/posts/monorepos.md:101]: These two are just fine. +1 for mentioning the ask for help.

**Review by lachmanfrantisek — APPROVED** [review]
Nice and quick. Thanks!

**Review by jpopelka — APPROVED** [review]
nice!

File: content/posts/monorepos.md (hunk 2 above)
Comment (lachmanfrantisek) [reply content/posts/monorepos.md:57]: Sadly, the failed status does not look so appealing but we don't need to spend so much time on that..;)

Comment (lachmanfrantisek) [reply content/posts/monorepos.md:57]: Thanks

**Pull request merged**